app/ide-desktop/lib/dashboard/src/entrypoint.ts

```diff
@@ -27,4 +31,5 @@ main.run({
   projectManagerUrl: null,
   ydocUrl: null,
   appRunner: testAppRunner.TestAppRunner,
+  queryClient: commonQuery.createQueryClient(config => new reactQuery.QueryClient(config)),
```

Comment:
we want this to be a vue query client as this will have to be passed to GUI2 as well - just because it is the cloud dashboard doesn't mean GUI2 doesn't run inside it ;)